backend/core/src/listings/listings.controller.ts

```diff
@@ -102,12 +103,15 @@ export class ListingsController {
 
   @Get(`:listingId`)
   @ApiOperation({ summary: "Get listing by id", operationId: "retrieve" })
-  @UseInterceptors(CacheInterceptor)
-  async retrieve(@Param("listingId") listingId: string): Promise<ListingDto> {
+  // @UseInterceptors(CacheInterceptor)
```

Comment:
Was this left commented out to debug something? Or did you mean to change the behavior?

Reply:
That's on purpose. I've added a TODO and a follow up ticket: https://github.com/bloom-housing/bloom/issues/1636